hrjobcontract/views/modalForm.html

```diff
@@ -560,33 +560,55 @@ <h4 class="modal-title">{{copy.title}}</h4>
                 </div>
             </uib-tab>
             <uib-tab heading="Leave">
+              <div ng-controller="FormLeaveCtrl">
```

Comment:
Please use the [`controller as`](https://toddmotto.com/digging-into-angulars-controller-as-syntax/) syntax:

```html
<div ng-controller="FormLeaveCtrl as form">
</div>
```